Make clearer where default search operates. (#417)
obsidianmd/obsidian-docs: en/Plugins/Search.md

## Search terms

> A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, Obsidian only searches notes and canvases, including their filenames.

Reviewer: ```suggestion
A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. Obsidian only searches the contents of notes and canvases.

> [!tip] Searching paths and filenames
> By default, you can only search the paths and filenames of notes and canvases. To search for a path or filename of any file in the vault, use the `path` or `file` operator.
```

> A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, the search is performed only over all markdown notes and canvases (both their filenames and their content).

Reviewer: ```suggestion
A search term is the word or phrase that you enter in the search field. Learning how to write search terms effectively can help you quickly find what you're looking for, even in large vaults. By default, Obsidian only searches notes and canvases, including their filenames.
```

According to the [style guide](https://help.obsidian.md/Contributing+to+Obsidian/Style+guide#Notes%2C+files%2C+and+folders), "note" implies Markdown.

| Search operator | Description                                                                                                                                                                                                          |
|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `file:`         | Find text in filename. All files in the vault are considered.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |

Reviewer: ```suggestion
| `file:`         | Find text in filename. Matches any file in the vault.<p/>Example: `file:.jpg` or `file:202209`.                                                                                                                                                     |
```

> | `path:`         | Find text in file path. All files in the vault are considered.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |

Reviewer: ```suggestion
| `path:`         | Find text in file path. Matches any file in the vault.<p/>Example: `path:"Daily notes/2022-07"`.                                                                                                                                                    |
```
